### PR TITLE
Update azure_rm_devtestlabvirtualmachine relate document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_devtestlabvirtualmachine relate document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_devtestlabvirtualmachine/azure_rm_devtestlabvirtualmachine_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_devtestlabvirtualmachine.py [hunk 1]
```diff
@@ -45,9 +45,10 @@
             - linux
     vm_size:
         description:
-            - A valid Azure VM size value. For example, 'Standard_D4'. The list of choices varies depending on the
-              subscription and location. Check your subscription for available choices. Required when creating a VM.
-            - "Available values can be found here: U(https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general)"
+            - A valid Azure VM size value. For example, C(Standard_D4).
+            - The list of choices varies depending on the subscription and location. Check your subscription for available choices.
+            - Available values can be found on this website, link U(https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-general).
+            - Required when creating a VM.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_devtestlabvirtualmachine.py]:
```suggestion
            - Required when I(state=present).
```

File: lib/ansible/modules/cloud/azure/azure_rm_devtestlabvirtualmachine_facts.py [hunk 2]
```diff
@@ -122,19 +123,19 @@
...
                     description:
-                        - Publisher.
+                        - The publisher of the gallery image.
                     returned: when created from gallery image
                     type: str
                     sample: Canonical
                 version:
                     description:
-                        - Version.
+                        - The version of the allery image.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_devtestlabvirtualmachine_facts.py]:
```suggestion
                        - The version of the gallery image.
```

**Comment by Fred-sun** [comment]
ready_for_review

**Pull request merged**